.github/workflows/prerelease.yaml

```diff
@@ -34,10 +34,13 @@ jobs:
       - name: Print varaibles
```

Comment:
Might as well edit this typo 😃

Reply:
It's fixed in https://github.com/neo4j/neo4j-genai-python/pull/54 so I'd rather keep it like this  here :)